### PR TITLE
[docs] Replace deprecated Constants.isDevice in different pages

### PR DESCRIPTION
**Pull request by wobsoriano**
# Why

`Constants.isDevice` is deprecated.

# How

Used `Device.isDevice` from `expo-device` instead.

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/notifications.md [hunk 1]
```diff
@@ -431,7 +431,7 @@ Returns a `Promise` that resolves to an object with the following fields:
 #### Fetching the Expo push token and uploading it to a server
 
 ```ts
-import Constants from 'expo-constants';
+import Device from 'expo-device';
```
Comment (Simek) [inline docs/pages/versions/unversioned/sdk/notifications.md:434]: It looks like to me that this import is not needed for the example code below, can you remove that?

**Review by Simek — CHANGES_REQUESTED** [review]
Hello @wobsoriano, thank you for the updates! 👍 

I have left only one suggestion, but besides that, this changeset LGTM.

File: docs/pages/versions/unversioned/sdk/notifications.md (hunk 1 above)
Comment (wobsoriano) [reply docs/pages/versions/unversioned/sdk/notifications.md]: Done!

**Review by Simek — APPROVED** [review]
Thanks, let's ship this! 🚀

**Pull request merged**